apps/services/auth-api/src/app/modules/user-profile/user-profile.controller.ts

```diff
@@ -16,10 +16,19 @@ import type { User } from '@island.is/auth-nest-tools'
 export class UserProfileController {
   constructor(private readonly userProfileService: UserProfileService) {}
 
+  @Scopes('@identityserver.api/authentication')
+  @Get()
+  @ApiOkResponse()
+  async userProfile(@CurrentUser() user: User): Promise<UserProfileDTO> {
+    return this.userProfileService.getUserProfileClaims(user)
+  }
+
   @Scopes('@identityserver.api/authentication')
   @Get('individual')
   @ApiOkResponse()
-  async findIndividual(@CurrentUser() user: User): Promise<IndividuaInfoDTO> {
-    return this.userProfileService.findIndividual(user)
+  async individualUserProfile(
```

Comment:
We should add the `@ApiOperation({ deprecated: true })`  decorator to indicate that this endpoint is deprecated and will be removed.